README.md

```diff
@@ -20,7 +20,7 @@ To write the Excel file, we use the excellent [PHPExcel](https://github.com/PHPO
 
 Install the package with [composer](http://getcomposer.org):
 
-    composer require codemix/yii2-excelexport *
+    composer require codemix/yii2-excelexport "*"
```

Comment:
@eyesoverrun I think we can remove that argument completely. Not sure, why I put it there.